rholang/src/main/scala/coop/rchain/rholang/interpreter/SystemProcesses.scala

```diff
@@ -105,6 +107,36 @@ object SystemProcesses {
           } yield ()
       }
 
+      def getErrorMessagePar(str: String) =
```

Comment:
Please add return type, and do so for all the functions you define from now on.

Reply:
done